applications/GeoMechanicsApplication/tests/cpp_tests/test_utilities/element_setup_utilities.h

```diff
@@ -13,8 +13,10 @@
 #pragma once
 
 #include "geometries/point.h"
+#include "includes/condition.h"
 #include "includes/element.h"
 
+#include <geo_aliases.h>
```

Comment:
Since this is not a system header, let's use double quotes rather than angular brackets:
```suggestion
#include "geo_aliases.h"
```

In addition, I would suggest to bundle it with the previous block of `#include`s (to have groups of Kratos `#include`s and system `#include`s).

Reply:
I'll check if I can change the setting in CLion for automatic including, changed now